pkg/workload/workload.go

```diff
@@ -771,6 +771,22 @@ func IsActive(w *kueue.Workload) bool {
 	return ptr.Deref(w.Spec.Active, true)
 }
 
+func IsPodsReadyStartCond(w *kueue.Workload) *metav1.Condition {
```

Comment:
A function with a name starting with `Is` should return bool, so something is off here. 

Maybe we can introduce a helper function `FindConditionWithReason`? It could already replace places like this in a follow up: https://github.com/kubernetes-sigs/kueue/blob/a70fd62ad7649877147a150afa92725e366a01f5/pkg/workload/workload.go#L725-L727

Reply:
Added the function